tree/tree/test/TTree.cxx

```diff
@@ -0,0 +1,42 @@
+#include "TFile.h"
+#include "TTree.h"
+#include "TBranch.h"
+#include "TRandom.h"
+
+#include "gtest/gtest.h"
+
+
+class TTreeTest : public ::testing::Test {
+protected:
+    virtual void SetUp()
+    {
+        auto random = new TRandom(836);
+        auto file = new TFile("TTree.root", "RECREATE");
+        auto tree = new TTree("tree", "A test tree");
+        tree->SetAutoFlush(500);
+        Double_t data = 0;
+        auto branch = tree->Branch("branch", &data);
+
+        for (Int_t ev = 0; ev < 1000; ev++) {
+            data = random->Gaus(100, 7);
+            tree->Fill();
+            if (ev == 100) {
```

Comment:
The same situation is here for the lines 23-25, we don't need to change basket size, we just want to test simple case.